apps/protocol-frontend/src/components/ProfileForm.tsx

```diff
@@ -191,7 +169,7 @@ const ProfileForm = () => {
                 <Button
                   variant="secondary"
                   type="submit"
-                  onClick={disconnectLinearOnClick}
+                  onClick={async () => await disconnectLinearUser()}
```

Comment:
You can call this as disconnectLinearUser. No need to create an async function. Same for the function below

Reply:
They are not assignable to onClick.